Source/src/importers/SceneImporter.cpp

```diff
@@ -26,7 +26,8 @@ void Hachiko::SceneImporter::Import(const char* path, YAML::Node& meta)
     Scene* temp_scene = new Scene();
     // Cant load a scene that is not imported yet
     const ResourceScene* scene = static_cast<const ResourceScene*>(Load(scene_uid));
-    temp_scene->Load(scene->scene_data);
+    constexpr bool meshes_only = true;
```

Comment:
why not just pass true, if we are not using this constant anywhere else

Reply:
For me its a bit more clear like that because we see the meaning of the true parameter this way. Also it is replaced by a true in compile time. I can remove it no problem